whois-api/src/main/resources/applicationContext-api.xml

```diff
@@ -12,7 +12,7 @@
 
     <bean class="net.ripe.db.whois.query.WhoisQueryConfig" />
     <import resource="applicationContext-update.xml"/>
-    <bean class="net.ripe.db.nrtm4.Nrtmv4Config" />
+<!--    <bean class="net.ripe.db.nrtm4.Nrtmv4Config" />-->
```

Comment:
if this is no needed I suggest removing it

Reply:
Fixed